scripts/repository.lic

```diff
@@ -1317,10 +1341,46 @@ elsif (cmd[0] =~ /^download$/i)
   if list.empty?
     echo "error: file not found"
   elsif list.length > 1
+    names = list.map { |row| row[0] }
     list.unshift(headers)
-    respond "\nMultiple files match: be more specific\n\n#{format_list.call(list)}\n\n"
+    _respond "\nMultiple files match: be more specific\n\n#{format_list.call(list, 'download')}\n\n"
+    if names.length != names.uniq.length
+      respond ""
+      _respond Lich::Messaging.msg_format("info", " You can specify a specific author by adding --author=bob to your command or game by adding --game=gs to the command. For example:")
+      _respond Lich::Messaging.msg_format("info", "  ;repo download awsomeScript --author=bob")
+      _respond Lich::Messaging.msg_format("info", "  ;repo download awsomeScript --game=gs")
+      respond ""
+    end
   else
-    echo "downloading #{list[0][fi]} in 3 seconds... (;k #{script.name} to cancel)"
+    if link_clicked
+      time_out = 10
+      start_time = Time.now
+      confirmed = false
+
+      respond ""
+      respond " You have clicked a link to download #{list[0][fi]} by the author #{list[0][headers.index('author')]}"
+      respond " To confirm this action please ';send yes' in the next #{time_out} seconds to confirm."
+      respond ""
+
```

Comment:
Below loop to be extracted as a get_user_confimration proc for reuse in other functions that need it (like delete/rate)